server/neptune/workflows/internal/prrevision/workflow.go

```diff
@@ -119,6 +150,14 @@ func (r *Runner) setRevision(ctx workflow.Context, req Request, prs []github.Pul
 	return nil
 }
 
+func (r *Runner) listModifiedFilesFuture(ctx workflow.Context, taskQueue string, req activities.ListModifiedFilesRequest) workflow.Future {
```

Comment:
Personally not a fan of having types in the function name.  Maybe something like listModifiedFilesAsync would be better.